rosbag2_py/src/rosbag2_py/_transport.cpp

```diff
@@ -262,6 +272,10 @@ PYBIND11_MODULE(_transport, m) {
     "start_offset",
     &PlayOptions::getStartOffset,
     &PlayOptions::setStartOffset)
+  .def_property(
+    "start_offset",
```

Comment:
It should be "playback_until" ?

Reply:
Done. Thanks for pointing it out. See 292ece7